tool/jenkins/jenkins_build.sh

```diff
@@ -109,7 +115,7 @@ docker exec madlib bash -c 'rm -rf /build_jenkins/src/ports/postgres/modules/uti
 docker exec madlib bash -c 'rm -rf /build_jenkins/src/ports/postgres/modules/mxgboost/test/madlib_xgboost.sql_in'
 docker exec madlib bash -c 'rm -rf /build_jenkins/src/ports/postgres/modules/kmeans/test/kmeans.sql_in'
 # Run dev check
-docker exec madlib bash -c '/build_jenkins/src/bin/madpack -s mad -p postgres  -c postgres/postgres@localhost:5432/postgres -d /tmp dev-check' | tee $workdir/logs/madlib_dev_check.log
+docker exec madlib bash -c '/build_jenkins/src/bin/madpack -s mad -p postgres  -c postgres/postgres@localhost:5432/postgres -d /tmp dev-check -t pmml' | tee $workdir/logs/madlib_dev_check.log
```

Comment:
We probably want to test the entire dev-check suite

Reply:
Yes this was only for debugging. I'll remove it